apps/src/templates/SchoolNotFound.jsx

```diff
@@ -51,38 +96,40 @@ export default class SchoolNotFound extends Component {
                 name="school_name_s"
                 value={this.props.schoolName}
                 onChange={this.handleChange.bind(this, "schoolName")}
-                style={styles.input}
+                style={inputStyle}
               />
             </label>
           </div>
-          <div style={styles.field}>
-            <label>
-              <div style={styles.question}>
-                {i18n.schoolType()}
-                <span style={styles.asterisk}> *</span>
-              </div>
-              <select
-                name="school_type_s"
-                value={this.props.schoolType}
-                onChange={this.handleChange.bind(this, "schoolType")}
-                style={styles.schoolNotFoundDropdown}
-              >
-                {schoolTypes.map((schoolType, index) =>
-                  <option
-                    value={schoolType}
-                    key={index}
-                  >
-                    {schoolType}
-                  </option>
-                )}
-              </select>
-            </label>
-          </div>
+          {this.props.schoolType !== 'omitted' &&
```

Comment:
Later: It might be nice to approach this differently, either with a constant `SchoolNotFound.OMITTED_FIELD` or a separate prop that is a list of omitted fields.